packages/atomic/src/components/atomic-refine-modal/atomic-refine-modal.pcss

```diff
@@ -1 +1,8 @@
 @import '../../global/global.pcss';
+
+/**
+ * @prop --atomic-refine-modal-facet-margin: The spacing between the modal's facets.
+ */
+.shadow-lg {
+  --tw-shadow: 0px -2px 8px rgba(229, 232, 232, 0.75);
```

Comment:
What is the difference between `--tw` prefixed CSS variables and `--atomic` prefixed variables?

Reply:
tailwind use variables for these effects as well. It was the only way to leverage the shadow by changing its direction for this component only